src/Time/Time.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * This file is part of the Zest Framework.
+ *
+ * @author   Muhammad Umer Farooq <lablnet01@gmail.com>
+ * @author-profile https://www.facebook.com/Muhammadumerfarooq01/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ * @since 3.0.0
+ *
+ * @license MIT
+ */
+
+namespace Zest\Time;
+
+use Zest\Contracts\Time\Time as TimeContract;
+
+class Time implements TimeContract
+{
+    /**
+     * Get "now" time
+     *
+     * @param string $timezone Valid php supported timezone.
+     *
+     * @since 3.0.0
+     *
+     * @return  int
+     */
+    public static function now($timezone = NULL)
+    {
+
+    }
+
+    /**
+     * Number of days in a month
+     *
+     * @param int $month A numeric month.
+     * @param int $year  A numeric year.
+     *
+     * @since 3.0.0
+     *
+     * @return  int
+     */
+    public static function daysInMonth($month = 1, $year = 1970)
+    {
+
+    }
+
+    /**
+     * Determine whether the year is leap or not
+     *
+     * @param int $year A numeric year.
+     *
+     * @since 3.0.0
+     *
+     * @return  bool
+     */ 
+    public static function isLeapYear($year = null)
+    {
+        if (null === $year) {
```

Comment:
If the changes are accepted that I mention on previous comment.

This `null` condition can be removed.